src/NuGet.Core/NuGet.Configuration/PublicAPI.Shipped.txt

```diff
@@ -413,6 +413,7 @@ static NuGet.Configuration.Settings.LoadSettingsGivenConfigPaths(System.Collecti
 static NuGet.Configuration.Settings.LoadSpecificSettings(string root, string configFileName) -> NuGet.Configuration.ISettings
 static NuGet.Configuration.SettingsUtility.DeleteConfigValue(NuGet.Configuration.ISettings settings, string key) -> bool
 static NuGet.Configuration.SettingsUtility.DeleteValue(NuGet.Configuration.ISettings settings, string section, string attributeKey, string attributeValue) -> bool
+static NuGet.Configuration.SettingsUtility.GetForceUpdatePackageLastAccessTimeEnabledStatus(NuGet.Configuration.ISettings settings) -> bool
```

Comment:
Same as the previous. All of this goes to unshipped.

Reply:
Indeed, fixed.